tests/drivers/spi/spi_loopback/boards/black_f407ve.conf

```diff
@@ -0,0 +1,4 @@
+CONFIG_SPI=y
+CONFIG_SPI_LOOPBACK_DRV_NAME="SPI_2"
+CONFIG_SPI_STM32_INTERRUPT=y
+CONFIG_SPI_LOOPBACK_MODE_LOOP=y
```

Comment:
I'm not familiar with this test, but shouldn't it  be done at project level?

Reply:
Hmm, I am not sure I understand your point. I can remove the SPI loopback test, it was just there that I used to verify that the SPI was working correctly.

Reply:
I meant the `CONFIG_SPI_LOOPBACK_MODE_LOOP=y` configuration.
Isn't it valid for all boards running the test  ?

Reply:
I removed the board from spi tests, it was just a leftover and should not have been there anyway.